src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -457,7 +340,6 @@ of `pan-x | pan-y | manipulation` you prevent the default double-tap
 behavior.
 
 This allows you to implement a double-tap gesture yourself.
-In the case of IE10+, it also eliminates the 300ms click delay.
 
 Below is a list of the available parameters for *touch-action*.
```

Comment:
Maybe just change the table from pretending to be an exhaustive list, to just listing the four most common scenarios.  I.e.:
- touch-action: none;    For surfaces where all touches are handled by JavaScript (maps, games, etc.)
- touch-action: pinch-zoom;   Like the above but leaves browser pinch-zoom enabled (eg. maybe your drag and drop example)
- touch-action: pan-y pinch-zoom;   Handle horizontal scrolls in JavaScript without disabling vertical scrolling or pinch-zooming (eg. image carousels).
- touch-action: manipulation;   Just disable double-tap to avoid the click delay, leave all scrolling and pinch-zooming up to the browser.

WDYT?